src/Lex.x

```diff
@@ -21,7 +21,6 @@ tokens :-
   behavior                              { mk BEHAVIOUR }
   of                                    { mk OF }
   interface                             { mk INTERFACE }
-  constructor                           { mk CONSTRUCTOR }
```

Comment:
Why was constructor removed?

Reply:
TL;DR: to unify identifiers and the `constructor` keyword

I didn't want to make constructor a reserved word because I wanted to be able to name constructor specs as for example
```
behaviour constructor of A
interface constructor()

...
```

but also as:
```
behaviour init of A
interface constructor()

...

```

which meant that I would need to have two different parsing rules, (either taking a `constructor` or an id), or just forget about `constructor` having a special meaning in the first place. It doesn't matter what you end up calling the interface that defines the constructor, what matters is only the arguments.

I've also been toying with the idea of calling initialization specs something different than `behaviour` altogether, but haven't found a good alternative yet.

Reply:
Ok!